modules/reference/pages/topic-properties.adoc

```diff
@@ -242,6 +246,44 @@ When `segment.bytes` is set to a positive value, it overrides the cluster proper
 
 ---
 
+[[writecaching]]
+==== write.caching
+
+The write caching mode to apply to a topic. 
+
+When `write.caching` is set to on, it overrides the cluster property xref:cluster-properties.adoc#write_caching[`write_caching`]. This caches batches of writes in memory until the segment appender chunk is full, instead of flushing for every `acks=all` write. Flushes follow <<flushms, `flush.ms`>> and <<flushbytes, `flush.bytes`>>, whichever is reached first. 
+
+**Default**: `off`
+
+**Values**:
+
+- `on` - Turns on write caching for a topic, according to size-based and time-based retention flushing.
+- `off` - Turns off write caching for a topic, according to size-based and time-based retention flushing.
+
+**Related topics**:
+
+- xref:develop:produce-data/configure-producers.adoc#write-caching[Write caching]
+
+---
+
+[[flushms]]
+==== flush.ms
+
+For <<writecaching,`write.caching`>>, this is the maximum delay (in ms) between two subsequent flushes. After this delay, the log is automatically force flushed.
+
+**Default**: `100`
+
+---
+
+[[flushbytes]]
+==== flush.bytes
+
+For <<writecaching,`write.caching`>>, this is the maximum bytes not flushed per partition. If this configured threshold is reached, the log is automatically flushed, even though it wasn't explicitly requested.
+
+**Default**: `262144`
```

Comment:
cluster level default is this configuration "raft_replica_max_pending_flush_bytes" (if no topic override exists).

Reply:
@bharathv Somehow `raft_replica_max_pending_flush_bytes` and `raft_replica_max_flush_delay_ms` never got documented in our [Cluster Properties](https://docs.redpanda.com/current/reference/cluster-properties/). Would you please send me the details for both of them, and I'll add?